chef/cookbooks/provisioner/recipes/update_nodes.rb

```diff
@@ -136,7 +136,7 @@
           hostname mnode.name
           macaddress mac_list[i]
           if mnode.macaddress == mac_list[i]
-            ipaddress admin_data_net.address
+            ipaddress admin_data_net.address unless admin_data_net.nil?
```

Comment:
Metrics/BlockNesting: Avoid more than 3 levels of block nesting. (https://github.com/bbatsov/ruby-style-guide#three-is-the-number-thou-shalt-count)